Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -74,8 +74,10 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
             textValueOfCellType[type.index] = hasValue ? value : (!isMandatory ? "" : nil)
             errorFlagOfCellType[type.index] = hasValue ? false : isMandatory
         }
-        if let country = country, country.iso3166Alpha2.isEmpty {
-            self.country = Country(iso3166Alpha2: Locale.current.regionCode ?? "", dialingCode: nil)
+        if country == nil,
```

Comment:
I would be tempted to remove this check, and since `country` is an optional, to just assign the result, unless we do something on `else` branch (we don't so far)